app/models/task.rb

```diff
@@ -3,39 +3,47 @@ class Task < ApplicationRecord
   belongs_to :user, optional: true
   belongs_to :instigator, class_name: "User", optional: true
 
-  enum status: [:accepted, :reassigned, :archived]
+  enum status: {
+    accepted: 0,
+    reassigned: 1,
+    archived: 2,
+    unassigned: 3,
+    assigned: 4,
+  }
```

Comment:
The order of the statuses haven't changed. The intention of the API is omit the statuses unless the values need to be explicitly mapped. Could you talk more about the motivations behind this patch?

Reply:
This one's interesting. I wanted to break the implicit dependency on the array order. I almost went and inserted the new ones at the start before I remembered about `enum`. I wanted to communicate that I knew that order was important here, if I were to reorder them (either by flow or alphabetically). In the end I didn't reorder them.

Reply:
Actioned in 82f6ef3.